todos/index.html

```diff
@@ -13,7 +13,7 @@
         <link rel="stylesheet" href="../styles/style.css" />
         <link rel="stylesheet" href="../styles/home.css" />
         <link rel="stylesheet" href="./todos.css" />
-        <title>Document</title>
+        <title>Todos Halfbaked</title>
```

Comment:
Nice work! I like how you added the title, as it is not required, but thorough! Awesome job!